ee/desktop/runner/runner.go

```diff
@@ -101,10 +103,10 @@ func WithProcessSpawningEnabled(enabled bool) desktopUsersProcessesRunnerOption
 	}
 }
 
-// WithGetter sets the key/value getter for agent flags
-func WithGetter(getter types.Getter) desktopUsersProcessesRunnerOption {
+// WithKnapsack sets the knapsack
+func WithKnapsack(knapsack *knapsack.Knapsack) desktopUsersProcessesRunnerOption {
```

Comment:
I'm fine with this, but I think `knapsack` is basically required, and should be a primary argument to `New`

Reply:
Agree, fixed.